FEATURE: add action to copy resource URI for a package resource

PRGfx opened the pull request:
Adds an entry to the "copy references" popup to copy the resource:// URI of a file to the clipboard.

For this I added a few utility function to the existing `ComposerUtil` to get the package key from a composer manifest.

cvette commented on src/main/java/de/vette/idea/neos/util/ComposerUtil.java:
```diff
@@ -88,4 +85,101 @@ public static Map<String, String> getNamespaceMappings(JsonFile composerFile)
+        var namespaceMapping = getNamespaceMappings(composerFile);
+        if (!namespaceMapping.isEmpty()) {
+            var firstNamespacePath = namespaceMapping.keySet().stream().findFirst();
+            var packageKeyFromNamespace = firstNamespacePath.get()
```
It should be checked with firstNamespacePath.isPresent() if it has a value, before accessing

PRGfx replied:
You mean instead of checking whether the Map is empty? Is there a java reasoning about one being better? I'm not fluent with java..
Doing both showed a notice that this would be unnecessary.

cvette replied:
namespaceMapping.isEmpty() should be enough I guess. Only paid attention to the notice for the get() call.

PRGfx commented on src/main/java/de/vette/idea/neos/actions/CopyResourceUri.java:
```diff
@@ -0,0 +1,102 @@
+
+public class CopyResourceUri extends CopyPathProvider {
+
+    String resourceUri = null;
```
I removed it

PRGfx commented:
I added tests for the package-key utility, which actually brought up an issue and fixed that.

When testing I noticed the problems with the org.jetbrains.intellij version that you since fixed, so I rebased onto recent main.
